cmd/short/short.go

```diff
@@ -63,30 +94,34 @@ func Relink(url_link string) (string, error) {
 		"url":url_link,
 	})
 	if err != nil {
-		log.Fatal(err)
+		return "", err
 	}
 	resp, err := http.Post("https://rel.ink/api/links/", "application/json", bytes.NewBuffer(jsn))
 	if err != nil {
-		log.Fatal(err)
+		return "", err
 	}
 	defer resp.Body.Close()
 	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
-		log.Fatalln(err)
+		return "", err
 	}
 	answer := make(map[string]interface{})
 	err = json.Unmarshal(body, &answer)
 	if err != nil {
-		log.Fatal(err)
+		return "", err
 	}
 	clean_url := "https://rel.ink/" + answer["hashid"].(string)
```

Comment:
Also, there are at least two possible errors that  not handled in code:
- Since you are unmarshalling to general `map[string]interface` there are no guarantees that field `hashid` will be present in answer at all.  So you need to you [comma, ok idiom](https://golang.org/doc/effective_go.html#maps) to read this field in map;
- Casting from `interface{}` to a string might cause error too if an underline type is not `string`.  You need to use [comma, ok for type assertion too](https://tour.golang.org/methods/15)